### PR TITLE
06 -  Add Order Form

### PR DESCRIPTION
**Pull request by Baavi**

### CODE REVIEW

File: src/components/Pages/addOrder.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+#order_h1 {
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 58px;
+}
+.submit_btn {
+  background: #75bd00;
+  border-radius: 7px;
+}
```
Comment (RoryHowlettUni) [inline src/components/Pages/addOrder.css]: * Currently none of this CSS selectors are referring to anything. The .class selectors should refer to classNames and the #class selectors refer to id attributes
* Font-size should use rem sizing. Rem refers to the font-size set in the root element (which is in index.css), so 1rem = 16px and so on. Just makes things more consistent
* Spacing (padding, margin, width and height) should also refer to rem sizing in most cases too
* In the root element Khang set up some global styles for the colours used throughout our webpage. So rather then using "color: #ffffff" you should use "color: var(--clr-white)"

File: src/components/Pages/addOrder.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          type={"email"}
+          placeholder={"alfie.santos@gmail.com"}
+        />
+        <Button buttonType={"primary"} isDisabled={true}>
+          Add Order
+        </Button>
+      </form>
+    </div>
+  );
+};
```
Comment (RoryHowlettUni) [inline src/components/Pages/addOrder.jsx:25]: * import CSS is commented out? Any particular reason for this?
* The placeholders don't match the ones set in Figma, may be worth double checking this
* You should swap ID to Name. InputField doesn't take an ID value and for the req.body.(thingy) I believe this uses the Name attribute to refer to the values submitted in the form. It's worth double checking the component to see what fields it excepts :)

Comment (RoryHowlettUni) [reply src/components/Pages/addOrder.jsx:25]: Also for the InputFields we are only using Date and Text types, so the Member ID field should just be Text. In Figma is does say to enter an email or mobile number, I feel like we've discussed about only using one or the other, could be discussing that with everyone again...

Comment (RoryHowlettUni) [reply src/components/Pages/addOrder.jsx:25]: Ah I see where you got the placeholders now, those are example user inputs, not the placeholders for the fields

**Review by ShiroKatake — APPROVED** [review]
Looks good! 💯

**Pull request merged**